### ISSUE TITLE
Atom packages not publish....

### ISSUE THREAD

**Issue by thiagolucio** [atom/atom#7496]
Not sure this is the right place for this Issue (if yes, sorry). I'm really trying to publish a package to atom but I can not ..Does anyone know why or went through this same problem?
![terminal](https://cloud.githubusercontent.com/assets/5756266/8394618/c3542c0a-1d17-11e5-92f7-b2ec6fac1d88.jpg)

**Comment by mnquintana** [comment]
Awesome to hear you're working on your first package! :sparkles: 

The error is telling you what's wrong – you need to create a Git repo first, and set it up on GitHub, before you can publish your package. Take a look at the link in the error message for more information: https://help.github.com/articles/create-a-repo/

(Also for future reference, this particular issue would belong in [atom/apm](https://github.com/atom/apm), but for support questions like this I'd recommend posting to [Discuss](https://discuss.atom.io/), Atom's official forum. GitHub is better for reporting bugs or requesting enhancements.)

**Comment by 50Wliu** [comment]
@mnquintana it looks like it actually is published on Github: https://github.com/thiagolucio/toolbar-iconshortcuts

@thiagolucio are you sure `.git` exists in the folder you're currently in (`ls -al`)?

**Comment by thiagolucio** [comment]
Thank you all for answers. Surely the problem is in my lack of knowledge itself. In fact I already have a repository in GIT (https://github.com/thiagolucio/toolbar-iconshortcuts) but nevertheless reading the published instructions understood that it should have a repository within the Atom (https://atom.io/users/thiagolucio/) and not the github (https://atom.io/packages/toolbar-iconshortcuts)
As the instructions say to publish the package generated from within the atom itself I understood that to get up direct package folder (~ / github / my-package /) located on my computer.
This is the confusion in my head. I will try to make operations using the "apm help publish" test and send the package directly from my github repository.
![difculdade](https://cloud.githubusercontent.com/assets/5756266/8396604/c0159d50-1d83-11e5-8c14-21d65bb16ef8.jpg)

**Comment by batjko** [comment]
For reference, this discussion has been taken to Discuss:
https://discuss.atom.io/t/apm-publish-minor-doesn-t-work/18134/6

It's a confusion about the correct procedure.

**Comment by thiagolucio** [comment]
Sorry to use this channel to my personal doubt.

50Wliu, I have a folder yes. You're right it, that guy is my repository in Git, you were smart to deduct.

![gitpaste](https://cloud.githubusercontent.com/assets/5756266/8396645/511691d2-1d85-11e5-88cb-0afd3a762572.jpg)
I opened a discussion
https://discuss.atom.io/t/apm-publish-minor-doesn-t-work/18134/3

those interested in helping, I am grateful. Anyway thank you all in advance. Closing the Issue.
